content/attacks/posts/Create - 2023-11-23-HecoCrossChainBridge&HTX.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2023-11-23
+target-entities: Heco Cross Chain Bridge and HTX
+entity-types:  
+   - Exchange
+   - Bridge
+attack-types:
+   - Private Key Leak
+title: “HECO Chain bridge compromised over $86.6M sent to suspicious addresses”
```

Comment:
Rephrase the title, term `suspicious address` seems unsuitable, cause hack is confirmed